packages/x-tree-view/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-tree-view",
-  "version": "7.0.0-alpha.1",
+  "version": "7.0.0-alpha.4",
```

Comment:
Version bump due to `@mui/base` upgrade in https://github.com/mui/mui-x/commit/200c7f7337d6220a65089b97ac2452e86b978de0#diff-e58865fd90960990fbfab6c723e16a26839669aebd39fafbefac5fc641889e49

Reply:
I thought that having `"@mui/base": "^5.0.0-beta.24"` as a dependency won't allow upgrading to `5.0.0-beta.26` because it's a prerelease, but it's not the case (https://github.com/npm/node-semver#prerelease-tags)
We can skip it 👍🏻